### ISSUE TITLE
Precision for SURF assignment is too tight

### ISSUE THREAD

**Issue by rmcdermo** [firemodels/fds#12367]
The input file is in the validation suite [here](https://github.com/firemodels/fds/blob/master/Validation/Convection/FDS_Input_Files/impinging_jet_Re_1e5_coarse.fds).

In the attached case, I stack three meshes on top of each other with DZ = 0.333333333333333.

This leads (in this case) to the middle section of meshes not being assigned a 'WALL' SURF type for the ceiling.  This occurs with both
```
&VENT DB='ZMAX', SURF_ID='WALL' /
```
and
```
&VENT PBZ=1, SURF_ID='WALL' /
```
Using `MB=1` is not desirable because it leads to fictitious Smokeview planes being drawn.

<img width="752" alt="Screenshot 2024-01-12 at 4 21 10 PM" src="https://github.com/firemodels/fds/assets/4418497/076e4146-8b5a-4769-85fa-75503877823a">

**Comment by mcgratta** [comment]
We were using the intrinsic function `SPACING(ZF)` to determine if the vent was outside the domain. This value is about 1.11e-16. I tried `TWO_EPSILON_EB`, which is about 4.44e-16. The difference between the upper boundary of the domain (1.0) and the upper boundary of the fine, inner mesh (0.999999999999999) is about 6.66e-16. I just decided to use `10*SPACING` for just this routine. We'll see if there are any unintended consequences.
